### PR TITLE
Create separate blindfold entry for the duct tape recipe.

### PR DESCRIPTION
**Pull request by Broken-Admin**

### CODE REVIEW

**Comment by Broken-Admin** [comment]
Sorry I such a dimwit.

**Comment by Broken-Admin** [comment]
Duct Tape is apparently made of [fabric & plastic](http://www.todayifoundout.com/index.php/2011/05/what-duct-tape-is-made-of/)

**Comment by Cyrano7** [comment]
Is duct tape even recoverable in deconstruction recipes? If it is it shouldn't be.

**Comment by Broken-Admin** [comment]
I was kind of questioning that myself. I'll fix that.

**Comment by Broken-Admin** [comment]
Whoops.

**Comment by Broken-Admin** [comment]
@Rivet-the-Zombie Could I get you to review this?

**Comment by BevapDin** [comment]
This should use `"copy-from": "blindfold"` and only set values explicitly that differ from the standard blindfold.

**Comment by Broken-Admin** [comment]
Hmmm, seems it needs more testing. I'll have to get on that.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -1063,6 +1063,12 @@
     "flags": [ "BLIND" ]
   },
   {
+    "copy-from": "blindfold",
+    "id": "blindfold_tape",
+    "type": "ARMOR",
```
Comment (Cyrano7) [inline data/json/items/armor.json]: You still need to have a name field copy-from entries.

File: data/json/recipes/armor/head.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "components": [ [ [ "felt_patch", 4 ] ] ]
   },
   {
-    "result": "blindfold",
+    "result": "blindfold_tape",
     "type": "recipe",
     "id_suffix": "from_tape",
```
Comment (Cyrano7) [inline data/json/recipes/armor/head.json]: You can delete this line now

**Comment by Cyrano7** [comment]
The recipe probably shouldn't be reversible like the normal one also.

**Comment by Broken-Admin** [comment]
Think I got rid of that. *_*

**Comment by Cyrano7** [comment]
@Broken-Admin You removed the deconstruction recipe but the recipe entry still has the reversible: true value.

**Comment by Cyrano7** [comment]
Maybe add a hint in the description of the item so people have a better idea what it is. The only indicator as of now its different is that the material is different and you can't disassemble it. People might think its a bug.

**Comment by Rivet-the-Zombie** [comment]
Yeah, it could use a different name and a modified description.

**Comment by Broken-Admin** [comment]
Gonna name it, *"Duct-Tape Blindfold"*

**Comment by Cyrano7** [comment]
@Broken-Admin No you haven't been shut out of the project.

**Comment by Cyrano7** [comment]
Whoops. Apparently there is a keyboard shortcut for close and comment? Apologizes for that. What I was trying to type was.

@Broken-Admin No you haven't been shut out of the project. This PR has only be available to re-review for 4 days. There are 48 other open PRs right now. Some of which might have a higher priority as bug fixes. At the time you submitted this around there were like almost 70 PRs. So I would advise being patient for now and someone will get to it eventually if its the right fit for the game.

**Comment by Broken-Admin** [comment]
That's fine, I'm just starting a bunch of projects for more weapons and other items so I'll be adding to the stack. Sorry about that!

**Comment by Broken-Admin** [comment]
Going to close this and Re-Fork but make branches.

**Pull request closed without merging**